<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>528</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>500</height>
        </size>
       </property>
       <property name="title">
        <string>工具栏</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnStartMark">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>开始</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEndMark">
          <property name="text">
           <string>结束</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAuto">
          <property name="text">
           <string>自动</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnManul">
          <property name="text">
           <string>人工</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCondition">
          <property name="text">
           <string>条件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLine">
          <property name="text">
           <string>连线</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClear">
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton:pressed 
{
    color: blue;
}
</string>
          </property>
          <property name="text">
           <string>清除</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRedo">
          <property name="text">
           <string>Redo</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="btnGroupTitle">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>节点明细</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="tabPosition">
             <enum>QTabWidget::North</enum>
            </property>
            <property name="tabShape">
             <enum>QTabWidget::Triangular</enum>
            </property>
            <property name="currentIndex">
             <number>4</number>
            </property>
            <property name="elideMode">
             <enum>Qt::ElideLeft</enum>
            </property>
            <widget class="QWidget" name="tab_5">
             <attribute name="title">
              <string>节点基本信息</string>
             </attribute>
             <widget class="QLabel" name="label_4">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>62</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>节点名称</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="txtNodeName">
              <property name="geometry">
               <rect>
                <x>80</x>
                <y>10</y>
                <width>211</width>
                <height>21</height>
               </rect>
              </property>
             </widget>
             <widget class="QPushButton" name="btnSaveNodeDetail">
              <property name="geometry">
               <rect>
                <x>310</x>
                <y>0</y>
                <width>114</width>
                <height>32</height>
               </rect>
              </property>
              <property name="text">
               <string>保存</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>前置任务</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>绑定服务:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="txtPreTaskSV"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>服务方法:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="txtPreTaskFunc"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>参数绑定:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="txtPreTaskParams"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QPushButton" name="btnSavePrevTask">
                  <property name="text">
                   <string>保存</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>任务服务</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_11">
                  <item>
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>绑定服务:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="txtNodeSV"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_12">
                  <item>
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>服务方法:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="txtNodeFunc"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_13">
                  <item>
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>参数绑定:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="txtNodeParams"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QPushButton" name="btnSaveNodeSV">
                  <property name="text">
                   <string>保存</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_3">
             <attribute name="title">
              <string>后置任务</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout_18">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_11">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_15">
                  <item>
                   <widget class="QLabel" name="label_10">
                    <property name="text">
                     <string>绑定服务:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="txtNodePostTaskSV"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_16">
                  <item>
                   <widget class="QLabel" name="label_11">
                    <property name="text">
                     <string>服务方法:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="txtNodePastTaskFunc"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_17">
                  <item>
                   <widget class="QLabel" name="label_12">
                    <property name="text">
                     <string>参数绑定:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="txtNodePostTaskParams"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <item>
                 <widget class="QPushButton" name="btnSavePostSV">
                  <property name="text">
                   <string>保存</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_4">
             <attribute name="title">
              <string>参数设置</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QTableWidget" name="tableWidget">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="autoFillBackground">
                 <bool>true</bool>
                </property>
                <property name="autoScrollMargin">
                 <number>0</number>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::SingleSelection</enum>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <property name="gridStyle">
                 <enum>Qt::SolidLine</enum>
                </property>
                <property name="rowCount">
                 <number>0</number>
                </property>
                <property name="columnCount">
                 <number>4</number>
                </property>
                <column>
                 <property name="text">
                  <string>变量名</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>绑定方法变量名</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>数据类型</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>变量类型</string>
                 </property>
                </column>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QPushButton" name="pushButton">
                  <property name="text">
                   <string>保存</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnAddRow">
                  <property name="text">
                   <string>添加</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_3">
                  <property name="text">
                   <string>删除</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
